src/test/kotlin/three/consulting/epoc/controller/EmployeeControllerIntegrationTest.kt

```diff
@@ -0,0 +1,77 @@
+package three.consulting.epoc.controller
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import org.assertj.core.api.Assertions
```

Comment:
You could import the `assertThat` method in the assertj Assertions module to ensure consistency between test classes by calling just `assertThat`.
```kotlin
import org.assertj.core.api.Assertions.assertThat
```